modules/loop_permissions/loop_permissions.module

```diff
@@ -222,42 +222,27 @@ function loop_permissions_secure_permissions($role) {
       'change layouts in place editing',
       'change own username',
       'create external_sources content',
-      'create index content',
```

Comment:
Shouldn't this be part of the previous PR (https://github.com/os2loop/profile/pull/113)?